### PR TITLE
Added death ui&Adjusted collision body size

### PR DESCRIPTION
**Pull request by yaalyy**
- Added player death ui
- Fixed some typos
- Adjusted collision body size of drop item
- Fixed bugs

### CODE REVIEW

File: scripts/GameManager.gd [hunk 1]
```diff
@@ -49,3 +49,25 @@ func _on_level_up_continue_game():
...
 
+
+# When the player is dead
+func _on_player_dead():
+	print("Player dead")
+	get_tree().paused = true
+	$GameUI.timer_stop()
+	$Dead.init()
+	
+func _on_dead_restart_game():
```
Comment (yaalyy) [inline scripts/GameManager.gd:61]: Hi, could u please take a look at this function to restart the round? I totally have no idea why my restart button cannot trigger this function.
Many thanks
@Si1w

File: scripts/dead.gd [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
+	pass
+	
+func init():
+	self.show()
+	dead_label.show()
+	restart_btn.show()
+
+func _on_restart_pressed():
```
Comment (yaalyy) [inline scripts/dead.gd:20]: here is the button to emit the signal to trigger respawn

**Review by yaalyy — COMMENTED** [review]
please look at my issues

File: scripts/GameManager.gd [hunk 3]
```diff
@@ -49,3 +49,25 @@ func _on_level_up_continue_game():
 	$now_enemies.delete_enemies()
 	pass # Replace with function body.
 
+
+# When the player is dead
+func _on_player_dead():
+	print("Player dead")
+	get_tree().paused = true
```
Comment (yaalyy) [inline scripts/GameManager.gd:55]: might because of this tree pause

**Comment by yaalyy** [comment]
Ready for review❤️
Updates: 

- Added player death ui, so that player can respawn to the the start of the round
- Fixed some typos
- Adjusted collision body size of drop item
- Fixed bugs

**Comment by yaalyy** [comment]
❤️❤️❤️❤️❤️❤️❤️

**Review by Si1w — APPROVED** [review]
all done

**Pull request merged**